conda_smithy/templates/azure-pipelines-win.yml.tmpl

```diff
@@ -50,57 +55,124 @@ jobs:
         New-Item -Path $batchPath -ItemType "file" -Force
 
         Set-Content -Value $batchcontent -Path $batchPath
-
         Get-ChildItem -Path $batchDir
-
         Get-ChildItem -Path ($batchDir + '\..')
+      displayName: Patch VC9
+    {% endif %}
 
-      condition: contains(variables['CONFIG'], 'vs2008')
-      displayName: Patch vs2008 (if needed)
+    {%- if 'vs2015' in config %}
+    # Set up conda since the vs2015 does not have a miniconda installation
+    - script: |
+        @"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
+      displayName: Install choco
+  
+    - powershell: |
+        Write-Host "##vso[task.prependpath]C:\ProgramData\chocolatey\bin"
+      displayName: Add choco to PATH
 
-    - task: CondaEnvironment@1
-      inputs:
-        packageSpecs: 'python=3.6 conda-build conda conda-forge::conda-forge-ci-setup=2' # Optional
-        installOptions: "-c conda-forge"
-        updateConda: false
-      displayName: Install conda-build and activate environment
+    - script: |
+        ECHO ON
+        choco install miniconda3 --params="'/D=D:\Miniconda3 /InstallationType=AllUsers'" --yes
+      displayName: Install miniconda3  
+        
+    - powershell: |
+        $content = Get-Content -Path C:\ProgramData\chocolatey\logs\chocolatey.log
+        foreach ($line in $content) {
+          Write-Host $line
+        }
+      displayName: Display miniconda3 installation log 
+    {% endif %}
 
-    - script: set PYTHONUNBUFFERED=1
+    {% if 'vs2017' in config %}
+    # Azure 2017 lacks the cmake piece needed for us to compiler successfully
```

Comment:
What does this mean? Can this be moved to a script in `conda-forge-ci-setup` like the vs2008 hacks we do in appveyor?

Reply:
Maybe, I have a pull request open to fix it in the image itself and then this section can go away.

Reply:
since we don't need this portion for if we get the new compilers, i'm going to remove it.  The rest of this pr is still useful i think for dealing with the other windows image types.